helix-term/src/health.rs

```diff
@@ -168,9 +204,13 @@ pub fn language(lang_str: String) {
                 .collect();
             if !suggestions.is_empty() {
                 let suggestions = suggestions.join(", ");
-                println!("Did you mean one of these: {} ?", suggestions.yellow());
+                writeln!(
+                    stdout,
+                    "Did you mean one of these: {} ?",
+                    suggestions.yellow()
+                )?;
             }
-            return;
+            return Err(std::io::Error::new(std::io::ErrorKind::Other, msg));
```

Comment:
We already print `msg` in red, having it return as the error string prints it twice

Reply:
Thank you. I have removed the line that returns the error. I am returning Ok now. The error is not printing twice anymore.